src/backend/core/malware_detection.py

```diff
@@ -0,0 +1,51 @@
+"""Malware detection callbacks"""
+
+import logging
+
+from django.core.files.storage import default_storage
+
+from lasuite.malware_detection.enums import ReportStatus
+
+from core.enums import DocumentAttachmentStatus
+from core.models import Document
+
+logger = logging.getLogger(__name__)
+
+
+def malware_detection_callback(file_path, status, error_info, **kwargs):
+    """Malware detection callback"""
+
+    if status == ReportStatus.SAFE:
+        logger.info("File %s is safe", file_path)
+        # Get existing metadata
+        s3_client = default_storage.connection.meta.client
+        bucket_name = default_storage.bucket_name
+        head_resp = s3_client.head_object(Bucket=bucket_name, Key=file_path)
+        metadata = head_resp.get("Metadata", {})
+        metadata.update({"status": DocumentAttachmentStatus.READY})
+        # Update status in metadata
+        s3_client.copy_object(
+            Bucket=bucket_name,
+            CopySource={"Bucket": bucket_name, "Key": file_path},
+            Key=file_path,
+            ContentType=head_resp.get("ContentType"),
+            Metadata=metadata,
+            MetadataDirective="REPLACE",
+        )
+        return
+
+    document_id = kwargs.get("document_id")
+    logger.error(
+        "File %s for document %s is infected with malware. Error info: %s",
+        file_path,
+        document_id,
+        error_info,
+    )
```

Comment:
I wonder if this is the proper logger level ? The dev team should act on something ? 
Also, should we use a dedicated logger name for this ? Like `django.security` so we can use it to send logs to another destination ?

Reply:
No the dev team can't act on something. You suggest to change it to info ?
We can use a dedicated name yes. `docs.security` ?

Reply:
```suggestion
    security_logger = logging.getLogger(f"docs.security.{__name__}")  # in case we want to add precise logging handler

    security_logger.warning(
        "File %s for document %s is infected with malware. Error info: %s",
        file_path,
        document_id,
        error_info,
    )
```

Warning because it is not a bug, but we might keep an eye on it. 
What do you think?

Reply:
I'm not sure about the logger name... maybe you can propose something better ^^ or just remove the `__name__` which might not be that helpful actually.